wuvt/trackman/tasks.py

```diff
@@ -14,6 +16,16 @@
 celery = make_celery(app)
 
 
+@periodic_task(run_every=crontab(day_of_week=1,hour=0, minute=0))
+def email_weekly_charts():
+    with app.app_context():
+        chart = db.session.query(Track.artist, Track.album,
+                func.count(Track.album)).filter(TrackLog.played >
+                        datetime.utcnow() -
+                        timedelta(days=7), TrackLog.new ==
+                        True).join(TrackLog).group_by(Track.album,Track.artist).order_by(desc(func.count(Track.album))).all()
```

Comment:
This is nitpicky, but I don't really like the inconsistent line lengths here. I would prefer to split this across multiple lines, especially when lines 22-25 have such little content split across multiple lines; it just looks weird.